cmd/zed/lake/serve/command.go

```diff
@@ -75,6 +75,10 @@ func (c *Command) Run(args []string) error {
 		return err
 	}
 	defer cleanup()
+	lakeRoot, err := c.lake.RootPath()
```

Comment:
Nit: No need for `lakeRoot`.
```suggestion
	c.conf.Root, err = c.lake.RootPath()
```